test-data/unit/check-kwargs.test

```diff
@@ -115,13 +115,20 @@ main:4: error: Unexpected keyword argument "b"
 [case testKeywordOnlyArguments]
 import typing
 def f(a: 'A', *, b: 'B' = None) -> None: pass
+def g(a: 'A', *, b: 'B') -> None: pass
 f(A(), b=B())
 f(b=B(), a=A())
 f(A())
 f(A(), B()) # E: Too many positional arguments for "f"
+g(A(), b=B())
+g(b=B(), a=A())
+g(A()) # E: Missing named argument "b" for function "g"
```

Comment:
Add test for multiple missing keyword-only arguments. Maybe also test a mix of optional and required keyword-only arguments?